kong/api/routes/server_names.lua

```diff
@@ -0,0 +1,16 @@
+local function method_not_allowed(self, db, helpers)
+  return helpers.responses.send_HTTP_METHOD_NOT_ALLOWED()
+end
+
+return {
+  -- GET / PATCH / DELETE /server_names/server_name are the only methods allowed
+
+  ["/server_names"] = {
+    before = method_not_allowed,
+  },
+
+  ["/server_names/:server_names/certificate"] = {
+    before = method_not_allowed,
```

Comment:
Same here, if this is the only:
/server_names/server_name

Endpoint, maybe we don't want to give reply that says that it is not allowed, users might try then other methods, and these seems no method allowed here. So I think it would be better to just have `404`.